src/deprecation.h

```diff
@@ -2,18 +2,22 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or https://www.opensource.org/licenses/mit-license.php .
 
+#include "consensus/params.h"
 #ifndef ZCASH_DEPRECATION_H
 #define ZCASH_DEPRECATION_H
 
 // Deprecation policy:
-// * Shut down 16 weeks' worth of blocks after the estimated release block height.
-// * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
+// (Per: https://github.com/zcash/zips/blob/master/zip-0200.rst)
+// Shut down nodes running this version of code, 16 weeks' worth of blocks after the estimated
+// release block height. A warning is shown during the 14 days' worth of blocks prior to shut down.
 static const int APPROX_RELEASE_HEIGHT = 698112;
-static const int WEEKS_UNTIL_DEPRECATION = 16;
-static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48);
+static const int RELEASE_TO_DEPRECATION_WEEKS = 16;
+static const int EXPECTED_BLOCKS_PER_HOUR = 3600/Consensus::POST_BLOSSOM_POW_TARGET_SPACING; // 48
```

Comment:
```suggestion
static const int EXPECTED_BLOCKS_PER_HOUR = 3600 / Consensus::POST_BLOSSOM_POW_TARGET_SPACING;
```

(In general, comments like this `// 48` are a bad idea because readers of the code will rely on the comment, when they should be relying on the code.)

Reply:
If I apply the reviewer's suggestions, should I add a comment prior to clicking "Resolve conversation"?

Reply:
Should I "click" "Resolve conversation" at all?  Or should I leave that to the reviewer who initiated it?

Reply:
Hmm...    maybe the PR is marked "Changes requested" until I click all the "Resolve conversation" buttons...